django_spanner/__init__.py

```diff
@@ -34,22 +33,24 @@
 if django.VERSION[:2] == (3, 2):
     USING_DJANGO_3 = True
 
-if USING_DJANGO_3:
-    from django.db.models.fields import (
-        SmallAutoField,
-        BigAutoField,
-    )
-    from django.db.models import JSONField
+USING_DJANGO_4 = False
+if django.VERSION[:2] == (4, 2):
+    USING_DJANGO_4 = True
+
+from django.db.models.fields import (
+    SmallAutoField,
+    BigAutoField,
+)
+from django.db.models import JSONField
 
 __version__ = pkg_resources.get_distribution("django-google-spanner").version
 
 USE_EMULATOR = os.getenv("SPANNER_EMULATOR_HOST") is not None
 
 # Only active LTS django versions (2.2.*, 3.2.*) are supported by this library right now.
-SUPPORTED_DJANGO_VERSIONS = [(2, 2), (3, 2)]
+SUPPORTED_DJANGO_VERSIONS = [(2, 2), (3, 2), (4, 2)]
 
 check_django_compatability(SUPPORTED_DJANGO_VERSIONS)
```

Comment:
can you help added a:
```
bypass_django_version_check = str(os.env.get('BYPASS_DJANGO_VERSION_CHECK', 'false')) == 'true'
if not bypass_django_version_check:
  check_django_compatability(SUPPORTED_DJANGO_VERSIONS)
```
so users can test django 5 with our own risk

Reply:
As mentioned earlier we cant do that. 
Please feel free to create a fork and make changes

Reply:
To provide a bit more context here: While it would be clear to everyone involved in this thread that such an option would be 'at your own risk and not supported', it might cause surprises for other people who discover this option and start using it. Especially if Django 5.x makes changes in the future that break something in combination with this library. As it currently has zero tests covering Django 5.x, it would not be noticed by anyone, and could cause failures in production.

Reply:
@olavloite I don't have a strong preference here, your point makes sense too.
We can make a folk and override this part of code to build our own Django 5 compatible version which is fine IMO.